### PR TITLE
Temporary Update for m.danawa.com, search.danawa.com/mobile/

### LINKED ISSUE #107: No products are appeared on the mobile version of danawa.com
**Issue Type**
* Bug Report

**Summary**
* There is a bug that no products are shown if `ssl.pstatic.net/adimg3.search/adpost/js/adpost_show_ads_v2.min.js` is blocked on the above two websites as shown below:

image

**Environment**
* All browsers using `YousList` (blocking `ssl.pstatic.net/adimg3.search/adpost/js/adpost_show_ads_v2.min.js`)

**Steps To Reproduce**
1. Access `m.danawa.com`
2. search for any product you want.

**Temporary solution**
* Makes an exception not blocking the script on two websites.

**Comments**
* Think this needs to be a *temporary fix* until we find a better one.

### PR DESCRIPTION
**Pull request by fenslett**
Temporarily fixed #107 the bug that no products are shown if the specific Javascript file is blocked on the above two websites.

### CODE REVIEW

File: youslist.txt [hunk 1]
```diff
@@ -18,6 +18,7 @@
 
 ! Exceptions
 @@||ad.smartmediarep.com/NetInsight/$domain=allvod.sbs.co.kr|news.sbs.co.kr|programs.sbs.co.kr
+@@||ssl.pstatic.net/adimg3.search/adpost/js/adpost_show_ads_v2.min.js$script,domain=search.danawa.com/mobile|m.danawa.com
```
Comment (yous) [inline youslist.txt]: I think `search.danawa.com/mobile` won't work for domain field.

```suggestion
@@||ssl.pstatic.net/adimg3.search/adpost/js/adpost_show_ads_v2.min.js$script,domain=search.danawa.com|m.danawa.com
```

**Comment by yous** [comment]
I can reproduce the problem on search.danawa.com/mobile/dsearch.php. But is there a problematic case on m.danawa.com?

**Comment by fenslett** [comment]
You can also see this bug on the category page like [this](http://m.danawa.com/product/productList.html?cateCode=12213298). See this image: image2

**Comment by yous** [comment]
Thanks! I get it. I'll merge this after the domain field has corrected.

File: youslist.txt (hunk 1 above)
Comment (fenslett) [reply youslist.txt]: You're correct. I found there's a mistake on that site, after checking it out.

**Comment by fenslett** [comment]
It wasn't possible to add commit and push to this PR branch didn't work, 
so I edited `search.danawa.com/mobile` to `search.danawa.com` and force-pushed the branch necessarily.

**Comment by yous** [comment]
Thanks!

**Pull request merged**